LICENCE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2025 Omnivector
+Copyright (c) `2025` `Omnivector, LLC`
```

Comment:
I think it's `Omnivector Corp`

Reply:
Got confirmation that it should be "Omnivector, LLC"